code/_onclick/adjacent.dm

```diff
@@ -84,7 +92,6 @@ Quick adjacency (to turf):
 	if(neighbor == loc || (neighbor.loc == loc)) return 1
 	if(!isturf(loc)) return 0
 	for(var/turf/T in locs)
-		if(isnull(T)) continue
```

Comment:
Any confidence in this being unneeded? Could do `if(T &&` below instead.

Reply:
locs is magical inbuilt BYOND list, and we do check loc to be turf before, so this should be always turfs